frontend/packages/knative-plugin/src/topology/components/nodes/EventSource.tsx

```diff
@@ -59,6 +64,24 @@ const EventSource: React.FC<EventSourceProps> = ({
   const { width, height } = element.getBounds();
   const size = Math.min(width, height);
   const { data, resources } = element.getData();
+  const allowEdgeCreation = useAllowEdgeCreation();
+  const isBootstrapServersPresent = resources?.obj?.spec?.bootstrapServers?.length > 0;
```

Comment:
we need to improve this instead of checking just for `bootstrapServers , should check if `bootstrapServers` is of `KC` i.e element is connected to `KC` via Kafka Connector